src/api/spec/controllers/webui/cloud/azure/upload_jobs_controller_spec.rb

```diff
@@ -0,0 +1,230 @@
+require 'rails_helper'
```

Comment:
I would put that into the commit where you implement the controller

Reply:
I didn't want to mess that commit. I splitted the tests too and there are too many changes at the same time for one commit from my POV

Reply:
Done already BTW